### ISSUE TITLE
zipFolder crashes for larger output

### ISSUE THREAD

**Issue by jase88** [NiklasPor/nx-remotecache-custom#9]
Hey,

Given a 33MB output in the dist folder, I receive the following error from *adm-zip*:
```
Unexpected error:
RangeError [ERR_OUT_OF_RANGE]: The value of "value" is out of range. It must be >= 0 and <= 4294967295. Received -333381632
    at new NodeError (node:internal/errors:371:5)
    at checkInt (node:internal/buffer:72:11)
    at writeU_Int32LE (node:internal/buffer:692:3)
    at Buffer.writeUInt32LE (node:internal/buffer:705:10)
    at Object.dataHeaderToBinary (/build/node_modules/adm-zip/headers/entryHeader.js:255:18)
    at Object.compressToBuffer (/build/node_modules/adm-zip/zipFile.js:257:49)
    at Object.toBuffer (/build/node_modules/adm-zip/adm-zip.js:780:25)
    at zipFolder (/build/node_modules/nx-remotecache-custom/create-remote-cache-store.js:12:16)
    at Object.store (/build/node_modules/nx-remotecache-custom/create-remote-cache-store.js:21:20) {
  code: 'ERR_OUT_OF_RANGE'
}
```

within zipFolder()
```
return zip.toBuffer();
```
causes this thrown error,

The out of range error is also documented here: https://github.com/cthackers/adm-zip/issues/375

Imho there are two options two "solve" this:
a) make the remote cache more stable against this error to at least make sure it is not exiting unexpectedly
b) use a different zip library

**Comment by NiklasPor** [comment]
#8 already suggests to switch to node-tar, so I might consider solving both issues at once.

**Comment by NiklasPor** [comment]
It should be fixed with `nx-remotecache-custom@2.0.0`. I'll update the other packages and then you can check if it's fixed.

**Comment by NiklasPor** [comment]
Should be now resolved with `nx-remotecache-azure@2.0.0`& `nx-remotecache-minio@2.0.0`. Happy hacking 👀

**Comment by jase88** [comment]
Wow, thank you very much!
